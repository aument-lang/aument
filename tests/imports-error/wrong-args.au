import "./wrong-args-1.au" as mod;
mod::x(1);