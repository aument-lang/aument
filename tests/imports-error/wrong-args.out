interpreter error(11) in -: wrong number of arguments for function 'x' (expected 0, got 1)
1 | import "./wrong-args-1.au" as mod;
